src/app/modules/upload-status/users-csv/users-csv.component.html

```diff
@@ -0,0 +1,158 @@
+<div class="dashboard-content">
+  <div class="c-toolbar">
+    <div class="row" style="padding-top: 5px;">
+      <a class="list-menu " [routerLink]="['/home']">
+        <mat-icon matTooltip="Home" class="home" aria-hidden="false" aria-label="Example home icon">keyboard_backspace
+        </mat-icon>
+      </a>
+
+      <div class="list-menu">
+        <mat-form-field class="custom-form-field">
+          <mat-label>Search </mat-label>
+          <input matInput #searchInput>
+          <button mat-icon-button matPrefix>
+            <mat-icon>search</mat-icon>
+          </button>
+        </mat-form-field>
+      </div>
+
+      <div class="list-menu ">
+        <mat-form-field class="custom-form-field">
+          <mat-label>Filter 1</mat-label>
+          <mat-select [(value)]="selected" (selectionChange)="selectorganisation()">
+            <mat-form-field class="custom-form-field">
+              <input style="padding-left: 10px;" matInput (keyup)="onKey($event.target.value)">
+            </mat-form-field>
+            <mat-option *ngFor="let state of organisations" [value]="state.value">{{state.label}}</mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+
+
+      <div class="list-menu ">
```

Comment:
Why is this filtering field repeated??

Reply:
That is static things, how should we handle?